src/v2/Components/BankDebitForm/BankDebitProvider.tsx

```diff
@@ -0,0 +1,71 @@
+import { FC, useEffect, useState } from "react"
+import { Appearance, loadStripe } from "@stripe/stripe-js"
+import { Elements } from "@stripe/react-stripe-js"
+import { getENV } from "v2/Utils/getENV"
+import { commitMutation, graphql } from "relay-runtime"
+import { createBankDebitSetupMutation } from "v2/__generated__/createBankDebitSetupMutation.graphql"
+import { useSystemContext } from "v2/System"
+import { BankDebitForm } from "./BankDebitForm"
+
+export const BankDebitProvider: FC = () => {
+  const { relayEnvironment } = useSystemContext()
+  const [clientSecret, setClientSecret] = useState("")
+
+  const stripePromise = loadStripe(getENV("STRIPE_PUBLISHABLE_KEY"), {
+    betas: ["us_bank_account_beta_2"],
```

Comment:
Maybe the beta flag can be dropped now the product is public.

Reply:
Good catch let me test it without.